integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -772,8 +773,11 @@ public void umount() {
       LOG.error("Unmounting {} when device is busy in reading/writing files. "
           + "{} fileInStream and {} fileOutStream remain open.",
           mMountPoint, mCreateFileEntries.size(), mOpenFileEntries.size());
+      if (!force) {
+        throw new FuseException("Timed out for umount due to device is busy.");
+      }
     }
-    super.umount();
+    super.umount(force);
```

Comment:
Why we pass the force to super class while the super class didn't use it at all?

Reply:
@maobaolong yes...super class doesn't need this param, but the Implementation class needs.